packages/dappkit/package.json

```diff
@@ -6,7 +6,7 @@
   },
   "dependencies": {
     "@celo/walletkit": "0.0.4",
-    "@celo/utils": "0.0.5",
+    "@celo/utils": "0.0.6-beta",
```

Comment:
This is unrelated. Right?
Please send a separate PR.

Reply:
I've added to "Other Changes" section in the PR.
It was necessary to fix this